src/app/components/home/home.component.ts

```diff
@@ -9,6 +9,41 @@ export class HomeComponent implements OnInit {
 
   constructor() { }
 
+  dummyData = [
```

Comment:
Please use DTO to convert the data even if its hardcoded

Reply:
Done that!, I've Added event model